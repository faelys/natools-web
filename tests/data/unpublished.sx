(elements
   (main-text "<p>Data in the unpublished page.</p>\n"
(render page-comments))
   (title "Unpublished Page"))
(tags (unpublished pages/news))
(comments
   (filter comment-filter)
   (backend comments unpublished)
   (tags comments))
(dates
   ("!publish" "2115-01-01T00:00:00Z"))
