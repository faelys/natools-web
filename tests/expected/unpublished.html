<html>
  <head>
    <title>Error 404</title>
  </head>
  <body>
    <h1>Error 404 - Not Found</h1>
    <p>/unpublished was not found on this server</p>
  </body>
</html>
